src/test/munit/redirect/redirect-test-case.xml

```diff
@@ -5,14 +5,16 @@
       xmlns:tls="http://www.mulesoft.org/schema/mule/tls"
       xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
       xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xmlns:java="http://www.mulesoft.org/schema/mule/java"
```

Comment:
why did you add this?

Reply:
I wanted to check an exception and I didn't remember to remove it.